reana_client/cli/code.py

```diff
@@ -94,7 +96,7 @@ def code_list(ctx, user, organization, workflow, filter, output_format):
         if output_format:
             click.echo(data.export(output_format))
         else:
-            click.echo(data)
+            click.echo(data.export('df'))
```

Comment:
The `df` export requires `pandas` which may be an unnecessarily big dependency... Perhaps use `tsv`? It does not look as pretty as the docker output though :smile: Perhaps we may massage ourselves and/or find another helper library that would pretty-print table-like dictionaries.